docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #79.            chunk_stop = chunk_start + m
</pre><p>This is different than the algorithm in the paper, but "similar" to what provided in the MATLAB code.</p><p> </p><p>In the line 10 of the Backward processing algorithm proposed in the paper, the start index of chunk gets updated to consider more subsequences; however, the chunk's stop index is fixed at i . In the MATLAB code, they have the optimized version where the algorithm avoids computing the distance between the query Q and the subsequences in the previously-considered chunk. To do this, the MATLAB code updates the stop index of chunk too. There are still some differences between the code here and the MATLAB code. </p><p> </p><pre class="ql-syntax" spellcheck="false">X_start = i-X+1+(expansion_num * SubsequenceLength);
X_end = i-(X/2)+(expansion_num * SubsequenceLength);
approximate_distance = min( real(MASS_V2(T(X_start:X_end), query))); 
       

# To update X and expansion_num at the end of each iteration in while-loop 
X = 2*X;
expansion_num = expansion_num+1;
</pre><p> </p><p>[Note to self]</p><p>Further analysis is needed to understand the reason behind using <code>expansion_num</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>